src/rocksdb.rs

```diff
@@ -1000,6 +1024,20 @@ impl Writable for DB {
     fn single_delete_cf(&self, cf: &CFHandle, key: &[u8]) -> Result<(), String> {
         self.single_delete_cf_opt(cf, key, &WriteOptions::new())
     }
+
+    fn delete_range(&self, begin_key: &[u8], end_key: &[u8]) -> Result<(), String> {
+        let batch = WriteBatch::new();
```

Comment:
There is no definition for `delete_range`:
https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L253